MSH/msh_elem.cpp

```diff
@@ -335,9 +335,9 @@ CElem::~CElem()
    Programing:
    06/2005 WW Implementation
 **************************************************************************/
-void CElem::FillTransformMatrix()
+void CElem::FillTransformMatrix(const bool reuse_matrix_cache)
```

Comment:
can you please add a description about the new argument? If I understand it correctly, the flag is used to recompute the transformation matrix. then, 'recompute_matrix' (or something like that) may be a better name.

Reply:
Added a description and changed the name.